circle.yml

```diff
@@ -13,4 +13,5 @@ test:
     - gem install --no-rdoc brakeman; bundle exec brakeman --exit-on-warn .
     - gem install --no-rdoc ruby_audit; bundle exec ruby-audit check
     - gem install --no-rdoc bundler-audit; bundle-audit update; bundle-audit check
+    - gem install --no-rdoc dawnscanner
```

Comment:
$$$

Once we have it downloaded in CI, we'll want to run it. Can you add a `bundle exec dawn --exit-on-warn .` to this, so the build runs it and fails if there's anything sketch?